### DISCUSSION TITLE
Where do they come from

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1396] [General]
Cannot figure out where these 44 files come from ( this is prior to importing tour from Edge)
![MTB](https://github.com/user-attachments/assets/b1c9b563-c80d-4b3c-b83b-75442d759ba9)
Here iare my settings:
![MTB2](https://github.com/user-attachments/assets/42b6ad32-51c1-432f-ae7e-428f5141a05d)

**Comment by FJBDev** [comment]
@gringojack 
Can you please make sure to read the [documentation ](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation)before asking questions ?

In this case, you can hover over the green folder "44" icon to see where they come from
See [here](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/get-tours/import-tours/import-tours-dashboard#action-import)
